src/start.sh

```diff
@@ -1,43 +1,9 @@
 #!/usr/bin/env bash
 
-export DEFAULT_VERSION=2.0.1
-FIRMWARE_PATH=/data/firmware/firmata-
-RELEASE_PATH=https://github.com/balena-io/balena-fin-coprocessor-firmata/releases/download/
-
 if ! test -f /data/firmware/bootloader.s37 ;
```

Comment:
Does this need to happen in the bash script? I mean its probably fine for now but we could move this to node to be cleaner?

Reply:
I meant the bootloader stuff?

Reply:
@curcuz

Reply:
aha! yes, I agree it does make sense to be moved into the js logic - mind filing an issue? I don't feel it should be blocking this PR